pkgs/by-name/ca/canon-capt/package.nix

```diff
@@ -0,0 +1,68 @@
+{
+  autoconf,
+  automake,
+  cups,
+  fetchurl,
+  stdenv,
+  unzip,
+}:
+let
+  version = "0.1.4.2-GxB";
+in
+stdenv.mkDerivation {
+  pname = "canon-capt";
+  inherit version;
+
+  src = fetchurl {
+    url = "https://github.com/mounaiban/captdriver/archive/8ecc3cde1ae9a20dcb015994bb0fea0dbefa2b83.zip";
+    sha256 = "sha256-QE9GiRC5ExH+LlooJ2eEFtsmA1aWzRoRfxh92EX6Tb8=";
+  };
+
+  nativeBuildInputs = [
+    autoconf
+    automake
+    unzip
+  ];
+
+  buildInputs = [ cups ];
+
+  # Fix for 'ppdc: Unable to find include file "<font.defs>"', which blocks '*.ppd' generation.
+  # Issue occurs in hermetic sandbox; this workaround is the current solution.
+  # Source: https://github.com/NixOS/nixpkgs/blob/9997402000a82eda4327fde36291234118c7515e/pkgs/misc/drivers/hplip/default.nix#L160
+  CUPS_DATADIR = "${cups}/share/cups";
+
+  unpackPhase = ''
+    runHook preUnpack
+    unzip $src
+    mv captdriver-*/* .
+    runHook postUnpack
+  '';
```

Comment:
You can probably skip all of this if you use fetchFromGitHub, it unpacks the source for you